libs/gi/page-team/src/CharacterDisplay/Content.tsx

```diff
@@ -151,7 +155,7 @@ function TabNav({
       )}
       <Tabs
         value={tab ?? 'setting'}
-        variant="fullWidth"
+        variant={isXs ? 'scrollable' : 'fullWidth'}
```

Comment:
This looks kinda funny on screens big enough to see all tabs and not trigger scroll view
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/c0087f61-b254-4cd5-87a3-d9c7ecc1caa8)
We used to have css to set the width of each tab to work around this, not sure if we still want that

Reply:
yeah its only gonna happen to screens <600px wide, not sure if its worth the effort to address